src/cdk/table/table.ts

```diff
@@ -1054,8 +1057,26 @@ export class CdkTable<T> implements AfterContentChecked, CollectionViewer, OnDes
     });
   }
 
+  private _initTableLayout() {
+    let layout: _TableLayout = null;
+    if (this._layoutStrategy) {
+      layout = this._isNativeHtmlTable
+          ? this._layoutStrategy.getNativeLayout(this)
+          : this._layoutStrategy.getFlexLayout(this);
+    }
+
+    if (!layout && this._isNativeHtmlTable) {
+      layout = this._applyNativeTableSections();
```

Comment:
It seems a little off to me that we use `null` as a way to signal to the table that it should fall back on a default. Should there not just be a default layout strategy that is always used, and that contains the native table document fragment builder?

Reply:
sgtm. I extracted the default layout into its own `_DefaultTableLayoutStrategy`. LMK what you think.